lib/Backend/arm/LowerMD.h

```diff
@@ -92,6 +92,9 @@ class LowererMD
             IR::Instr *     LowerLdEnv(IR::Instr *instr);
             IR::Instr *     LowerLdSuper(IR::Instr * instr, IR::JnHelperMethod helperOpCode);
             IR::Instr *     GenerateSmIntPairTest(IR::Instr * instrInsert, IR::Opnd * opndSrc1, IR::Opnd * opndSrc2, IR::LabelInstr * labelFail);
+#if DBG
+    static  void            GenerateDebugBreak( IR::Instr * insertInstr );
```

Comment:
Good catch, I've fixed this (and rebased to catch up to master) with the new commit.